### PR TITLE
Fixed controller cover notification

### PR DESCRIPTION
**Pull request by johnch18**
Made the controller cover less overbearing with switching off while still keeping functionality. Added toggle method to check boxes.

### CODE REVIEW

**Comment by matt-159** [comment]
Is there a reason BaseMetaTileEntity got completely reformatted?

**Comment by johnch18** [comment]
> Is there a reason BaseMetaTileEntity got completely reformatted?

Probably because I hit the key combination for it out of habit. Anything I can do to fix that?

**Comment by basdxz** [comment]
Manually is the only way here. 

Rather not have stuff be reformatted at random and it does make it bloody hard to see what actually changed here too

File: src/main/java/gregtech/common/covers/GT_Cover_ControlsWork.java
Comment (matt-159) [inline src/main/java/gregtech/common/covers/GT_Cover_ControlsWork.java:1]: This whole class is riddled with odd ternary statements and magic constants. I'm sure this makes sense to you but at a quick inspection, it's meaningless to me

**Review by matt-159 — CHANGES_REQUESTED** [review]

**Review by matt-159 — APPROVED** [review]

**Pull request merged**